lib/num_option.fz

```diff
@@ -67,10 +67,10 @@ is
   #
   public val T
     pre
-      safety: (num_option.this??)
+      safety: num_option.this??
   =>
     num_option.this ? v T => v
-                    | nil => fuzion.std.panic "num_option.val called on nil"
+                    | nil => fuzion.std.panic "num_option.val called on nil. You either turned off safety or you discovered a severe compiler bug! This code should be unreachable."
```

Comment:
I would not ever speculate we have a compiler bug, developers would do this anyway. My suggestion for this and the other messages is something like:

```suggestion
                    | nil => fuzion.std.panic "num_option.val called on nil. Enable `safety` to obtain a precondition failure for debugging."
```

Reply:
@fridis Yes, that makes more sense.